connectors/aws/aws-sns/element-templates/aws-sns-inbound-boundary.json

```diff
@@ -141,12 +141,11 @@
     "id" : "messageTtl",
     "label" : "Message TTL",
     "description" : "Time-to-live for the message in the broker (ISO-8601 duration)",
-    "optional" : false,
-    "value" : "PT0S",
+    "optional" : true,
     "constraints" : {
-      "notEmpty" : true,
+      "notEmpty" : false,
```

Comment:
this means `messageTtl` won't be in the XML at all, just wanted to highlight this @chillleader

Reply:
@johnBgood is that the expected behaviour?